app/src/main/kotlin/com/simplemobiletools/calculator/helpers/MoneyCalculatorImpl.kt

```diff
@@ -1,11 +1,12 @@
 package com.simplemobiletools.calculator.helpers
 
-import android.content.Context
-import com.simplemobiletools.calculator.R
-import com.simplemobiletools.calculator.operation.TaxOperation
-import com.simplemobiletools.calculator.operation.TipOperation
+import com.simpletools.calculator.commons.helpers.Calculator
+import com.simpletools.calculator.commons.R
+import com.simpletools.calculator.commons.operations.TaxOperation
+import com.simpletools.calculator.commons.operations.TipOperation
+import com.simpletools.calculator.commons.helpers.Formatter
 
-class MoneyCalculatorImpl(calculator: Calculator,taxCalculator: TaxCalculator, val context: Context) {
```

Comment:
Wondering why this Context parameter is being removed in a bunch of places. Was it unnecessary?

Reply:
Was mostly there for the widget. Made a dependency for the test harness. Couldn't find a good way to break it while keeping the context. I opted to removed the widget as it was legacy and all the other instances of calculator did not need it. Simpler code, lower coupling.